templates/tfengine/components/resources/compute_networks/main.tf

```diff
@@ -31,7 +31,7 @@ module "{{$resource_name}}" {
       subnet_ip              = "{{.ip_range}}"
       subnet_region          = "{{get . "compute_region" $.compute_region}}"
       subnet_flow_logs       = true
```

Comment:
Hmm.. even subnet_flow_logs is always true so the value in the config seems to be ignored